openapi.yml

```diff
@@ -32,10 +52,10 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/OrderFull"
-  /api/v1/nonce:
+  /api/v1/tip:
     get:
-      summary: The nonce to pick for a new order.
-      description: The best nonce to pick depends on how much time there is left in the current settlement according to the operator. This time is token-pair specific which is why the sell and buy tokens must be supplied.
+      summary: The tip that is used to pay solvers.
+      description: The tip is calculated by the order service and depends on the current gas price and the expected gas costs to settle the trade
       parameters:
         - name: sellToken
```

Comment:
Why does the tip requires a buy token? In my thinking, It should be only be relative to the sell token. Basically a client would ask "If sell GNO how much GNO you need as a TIP"? 

If there's a decision to take, like there's a minimal sell amount, we could model in the return data. "The GNO TIP is 0.1 GNO, but only for trades selling less than 10 GNO". Or any other logic.
So I wouldn't pass even the amounts.

I would suggest:
`/api/v1/tokens/`{sellToken}`/required-tip`

Reply:
Also, I think the operator should return also a date together with the tip (`expirationDate`). So the clients know until when the operator would allow to receive orders for that sell token with that minimal tip.

Reply:
Yeah I was thinking that we might want to subsidize differently, depending on the tokenpair and ordertype. But you are right, it is much easier in your suggestion and we should go for it.

Reply:
Okay cool! one big reason would be, that if we use the amounts, we would need to ask on every amount change, and the UX could feel a bit sluggish also we would be making many queries to the backend that could be avoided.